tests/internal/network/test_server.py

```diff
@@ -60,17 +61,17 @@ def test_create_server_socket_sets_up_listener(self, create_socket):
         sock = create_socket.return_value
 
         network.Server.create_server_socket(
-            self.mock, sentinel.host, sentinel.port)
+            self.mock, str(sentinel.host), sentinel.port)
         sock.setblocking.assert_called_once_with(False)
-        sock.bind.assert_called_once_with((sentinel.host, sentinel.port))
+        sock.bind.assert_called_once_with((str(sentinel.host), sentinel.port))
```

Comment:
Could probably do with a version of this for the unix case also since `bind()` is in both code paths.